### PR TITLE
Update configuation-in-mattermost-database.rst

### PR DESCRIPTION
**Pull request by Sayanta66**
Fixed the document with proper punctuation and other fixes, Also, updated the two links

### CODE REVIEW

File: source/configure/configuation-in-mattermost-database.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Configuration in the Mattermost Database
 ========================================
 
-A new configuration option was added in the `5.10 release <https://docs.mattermost.com/administration/changelog.html#configuration-in-database>`_ to use the database as the single source of truth for the active configuration of your Mattermost installation. This changes the Mattermost binary from reading the default ``config.json`` file to reading the configuration settings stored within a configuration table in the database.
+A new configuration option was added in the `5.10 release <https://docs.mattermost.com/install/self-managed-changelog.html#configuration-in-database>`_ to use the database as the single source of truth for the active configuration of your Mattermost installation. This changes the Mattermost binary from reading the default ``config.json`` file to reading the configuration settings stored within a configuration table in the database.
```
Comment (cwarnermm) [inline source/configure/configuation-in-mattermost-database.rst:4]:
```suggestion
A new configuration option was added in the `5.10 release <https://docs.mattermost.com/install/self-managed-changelog.html#configuration-in-database>`__ to use the database as the single source of truth for the active configuration of your Mattermost installation. This changes the Mattermost binary from reading the default ``config.json`` file to reading the configuration settings stored within a configuration table in the database.
```

File: source/configure/configuation-in-mattermost-database.rst [hunk 2]
```diff
@@ -157,13 +157,13 @@ The command to migrate the config to the database should always be run as the *m
    bin/mattermost config migrate ./config/config.json 'mysql://mmuser:mostest@tcp(127.0.0.1:3306)/mattermost?charset=utf8mb4,utf8&writeTimeout=30s'
 
 .. warning::
-   When migrating config, Mattermost will incorporate configuration from any existing ``MM_*`` environment variables set in the current shell.  See `Environment Variables  <https://docs.mattermost.com/administration/config-settings.html#configuration-settings>`_
+   When migrating config, Mattermost will incorporate configuration from any existing ``MM_*`` environment variables set in the current shell.  See `Environment Variables  <https://docs.mattermost.com/configure/configuration-settings.html#configuration-settings>`_
```
Comment (cwarnermm) [inline source/configure/configuation-in-mattermost-database.rst:160]:
```suggestion
   When migrating config, Mattermost will incorporate configuration from any existing ``MM_*`` environment variables set in the current shell.  See `Environment Variables  <https://docs.mattermost.com/configure/configuration-settings.html#environment-variables>`_
```

Comment (cwarnermm) [reply source/configure/configuation-in-mattermost-database.rst:160]: I've provided the updated URL.

File: source/configure/configuation-in-mattermost-database.rst [hunk 3]
```diff
@@ -157,13 +157,13 @@ The command to migrate the config to the database should always be run as the *m
...
    
-As with the environment file you'll have to escape any single quotes in the database connection string. Also, any existing SAML certificates will be migrated into the database as well so they are available for all servers in the cluster.
+As with the environment file, you'll have to escape any single quote in the database connection string. Also, any existing SAML certificates will be migrated into the database as well, so they are available for all servers in the cluster.
```
Comment (cwarnermm) [inline source/configure/configuation-in-mattermost-database.rst:162]:
```suggestion
As with the environment file, you'll have to escape any single quotes in the database connection string. Also, any existing SAML certificates will be migrated into the database as well, so they are available for all servers in the cluster.
```

Comment (cwarnermm) [reply source/configure/configuation-in-mattermost-database.rst:162]: In this context, the term "quotes" should remain plural.

File: source/configure/configuation-in-mattermost-database.rst [hunk 4]
```diff
@@ -157,13 +157,13 @@ The command to migrate the config to the database should always be run as the *m
...
 
-With configuration in the database enabled, any changes to the configuration are recorded to the ``Configurations`` and ``ConfigurationFiles`` tables. Furthermore, ``ClusterSettings.ReadOnlyConfig`` is ignored, enabling full use of the System Console.
+With the configuration in the database enabled, any changes to the configuration are recorded to the ``Configurations`` and ``ConfigurationFiles`` tables. Furthermore, ``ClusterSettings.ReadOnlyConfig`` is ignored, enabling full use of the System Console.
```
Comment (cwarnermm) [inline source/configure/configuation-in-mattermost-database.rst:164]:
```suggestion
When configuration in the database is enabled, any changes to the configuration are recorded to the ``Configurations`` and ``ConfigurationFiles`` tables. Furthermore, ``ClusterSettings.ReadOnlyConfig`` is ignored, enabling full use of the System Console.
```

**Review by cwarnermm — COMMENTED** [review]
Thanks for your docs contribution! I've provided feedback inline.

**Comment by justinegeffen** [comment]
Thank you for your PR. Unfortunately as the source repo has been deleted, we're not able to merge these changes. We really appreciate your contribution! :)

**Pull request closed without merging**